bedrock/firefox/firefox_details.py

```diff
@@ -246,13 +249,13 @@ def get_download_url(self, channel, version, platform, locale,
                 suffix = 'latest'
 
             _version = ('beta-' if channel == 'beta' else '') + suffix
-        elif not funnelcake_id:
+        elif not _include_funnelcake_param:
             # Force download via SSL. Stub installers are always downloaded via SSL.
             # Funnelcakes may not be ready for SSL download
             _version += '-SSL'
 
         # append funnelcake id to version if we have one
-        if funnelcake_id:
+        if _include_funnelcake_param:
```

Comment:
As long as downloads never break I'm happy. Having the "dumber" buttons that just link to scene 2 avoid adding the "f" param to non-win platforms seems more complex. Requesting the funnelcake at scene2 seems fine as long as scene2 is smart enough, and it sounds like it is.